configs/config.go

```diff
@@ -5,12 +5,10 @@ import (
 	"os"
 )
 
-// ConfigModel ...
 type ConfigModel struct {
```

Comment:
createFromEnvs and Validate could be merged by utilizing the `stepconf` package.